pkg/fakerp/client/config.go

```diff
@@ -46,5 +46,11 @@ func NewConfig(log *logrus.Entry) (*Config, error) {
 		return nil, fmt.Errorf("must set AZURE_REGIONS to a comma separated list")
 	}
 	log.Infof("using region %s", c.Region)
+	if c.WorkspaceResourceID == "" {
```

Comment:
IMHO this should be turned on in the environment so not all dev clusters have it on.

Reply:
I think all dev clusters should have monitoring enabled by default because:
- It gives the SREs confidence that the monitoring integration is working similarly for the customers
- Clusters created with a service such as ours almost always have native monitoring enabled out of the box (AKS, EKS, GKE etc)